app-backup/backuppc/backuppc-4.4.0.ebuild

```
@@ -0,0 +1,187 @@
+		myconf="${myconf} --bin-path nmblookup=$(type -p nmblookup)"
+	fi
+
+	/usr/bin/env perl ./configure.pl \
+		--batch \
+		--bin-path perl=$(type -p perl) \
+		--bin-path tar=$(type -p tar) \
+		--bin-path rsync=$(type -p rsync) \
+		--bin-path ping=$(type -p ping) \
+		--bin-path df=$(type -p df) \
+		--bin-path ssh=$(type -p ssh) \
+		--bin-path sendmail=$(type -p sendmail) \
+		--bin-path hostname=$(type -p hostname) \
+		--bin-path gzip=$(type -p gzip) \
+		--bin-path bzip2=$(type -p bzip2) \
+		--config-dir "${CONFDIR}" \
+		--install-dir /usr \
+		--data-dir "${DATADIR}" \
+		--hostname 127.0.0.1 \
+		--uid-ignore \
+		--dest-dir "${D}" \
+		--html-dir "${CGIDIR}"/image \
+		--html-dir-url /image \
+		--cgi-dir "${CGIDIR}" \
+		--fhs \
+		${myconf} || die "failed the configure.pl script"
+
+	ebegin "Installing documentation"
+
+	pod2man \
+		-errors=none \
+		--section=8 \
+		--center="BackupPC manual" \
+		"${S}"/doc/BackupPC.pod backuppc.8 \
+		|| die "failed to generate man page"
+
+	doman backuppc.8
+
+	# Place the documentation in the correct location
+	dodoc "${D}/usr/share/doc/BackupPC/BackupPC.html"
+	dodoc "${D}/usr/share/doc/BackupPC/BackupPC.pod"
+	rm -rf "${D}/usr/share/doc" || die
+
+	eend 0
+
+	# Setup directories
+	dodir "${CONFDIR}/pc"
+
+	keepdir "${CONFDIR}"
+	keepdir "${CONFDIR}/pc"
+	keepdir "${DATADIR}"/{trash,pool,pc,cpool}
+	keepdir "${LOGDIR}"
+
+	ebegin "Setting up init.d/conf.d/systemd scripts"
+	newinitd "${S}"/systemd/init.d/gentoo-backuppc backuppc
+	newconfd "${S}"/systemd/init.d/gentoo-backuppc.conf backuppc
+	systemd_dounit "${FILESDIR}/${PN}.service"
+
+	insinto "${APACHE_MODULES_CONFDIR}"
+	doins "${FILESDIR}"/99_backuppc.conf
+
+	# Make sure that the ownership is correct
+	chown -R backuppc:backuppc "${D}${CONFDIR}" || die
+	chown -R backuppc:backuppc "${D}${DATADIR}" || die
+	chown -R backuppc:backuppc "${D}${LOGDIR}"  || die
+}
+
+pkg_postinst() {
+	elog "Installation finished, you may now start using BackupPC."
+	elog
+	elog "- Read the documentation in /usr/share/doc/${PF}/BackupPC.html"
+	elog "  Please pay special attention to the security section."
+	elog
+	elog "- You can launch backuppc by running:"
+	elog
+	elog "    # /etc/init.d/backuppc start"
+	elog
+	elog "- To enable the GUI, first edit ${ROOT}/etc/conf.d/apache2 and add,"
+	elog
+	elog "    \"-D BACKUPPC -D PERL -D MPM_ITK\""
+	elog
+	elog "  to the APACHE2_OPTS line."
+	elog
+	elog "  Then you must edit ${ROOT}/etc/apache2/modules.d/00_mpm_itk.conf"
+	elog "  and adjust the values of LimitUIDRange/LimitGIDRange to include"
+	elog "  the UID and GID of the backuppc user."
+	elog
+	elog "  Finally, start apache:"
+	elog
+	elog "    # /etc/init.d/apache2 start"
+	elog
+	elog "  The web interface should now be running on,"
+	elog
+	elog "    http://127.0.0.1:8080/"
+	elog
+
+	# Generate a new password if there's no auth file
+	if [[ ! -f "${CONFDIR}/users.htpasswd" ]]; then
+		adminuser="backuppc"
+		adminpass=$( makepasswd --chars=12 )
```

Nit (style choice pretty much):
```suggestion
		adminpass=$(makepasswd --chars=12)
```